config.yml.sample

```diff
@@ -17,6 +17,7 @@ cache:
   quota: 90
   expiry: 30
 
+err_log_dir: /mnt/errs
```

Comment:
is this agreed?

Reply:
No, this is a placeholder - will check and update PR.